README.md

```diff
@@ -26,20 +26,17 @@ This will then produce the folders `zig-cache` (for temporary files) and `zig-ou
 To build `svg2tvgt`, go into the folder `src/tools/svg2tvgt` and do this:
 
 ```sh-session
-[user@host sdk]$ cd src/tools/svg2tvgt/
```

Comment:
i think this line is still necessary in the new version ^^

Reply:
Fixed!